.github/CODEOWNERS

```diff
@@ -965,6 +965,9 @@
 # Reviewers to double check any API changes
 /sdk/**/api/                                                       @KrzysztofCwalina @tg-msft
 
+# PRLabel: %Extension - WCF
```

Comment:
Please move this up to with the other non-management services, preserving alphabetical order by tag name.

Reply:
I have moved this to # ######## Services ########. Please let me know if this is still not the right place.